com.enjoyingfoss.Feeel.yml

```diff
@@ -0,0 +1,46 @@
+app-id: com.enjoyingfoss.feeel
```

Comment:
this file is a duplicate.

Reply:
Whoops — thanks for letting me know!

It seems that macOS has ignorecase set to true by default, and even when disabling it, renaming the file makes git act really strange. Come to think of it, I think the strange behavior might have come from my cloud storage syncing.

Anyway, hopefully it's alright now.